pyodide_build/pywasmcross.py

```diff
@@ -212,6 +226,20 @@ def handle_command(line, args, dryrun=False):
             return
         if arg.startswith('/tmp'):
             return
+        # automake generated configure scripts:
+        if 'DUALCASE' in env and 'as_nl' in env:
```

Comment:
Maybe add a link to https://www.gnu.org/software/autoconf/manual/autoconf-2.61/html_node/Special-Shell-Variables.html in the comment